experimental/oak_baremetal_launcher/src/main.rs

```diff
@@ -58,6 +58,10 @@ struct Args {
     #[clap(subcommand)]
     mode: Mode,
 
+    /// Consistent response size that the runtime should apply
+    #[clap(long, default_value = "1024")]
```

Comment:
Personally I would have left the default to 0, and also emit an error at startup if it's 0, effectively forcing the caller to specify it.